torch/csrc/jit/codegen/cuda/ir_interface_nodes.h

```diff
@@ -326,6 +326,16 @@ class TORCH_CUDA_API TensorView : public Val {
     return memory_type_;
   }
 
+  void setMemoryType(MemoryType mt) {
+    memory_type_ = mt;
+    bool is_inp_or_out =
```

Comment:
1. do we really need this local variable? it seems we can just inline it in the if condition?

2. can you please remove `this->` ? (yes, I know it was already in there)

Reply:
done